packages/indexer/src/handlers/events/evmHost/postRequestTimeoutHandled.event.handler.ts

```diff
@@ -67,6 +74,18 @@ export const handlePostRequestTimeoutHandledEvent = wrap(async (event: PostReque
 					blockTimestamp,
 				)
 				await VolumeService.updateVolume(`Transfer.${symbol}`, amountValueInUSD, blockTimestamp)
+
+				// Contract address volumes
+				const [fromIsContract, toIsContract] = await Promise.all([
+					isContractAddress(from, chain),
+					isContractAddress(to, chain),
+				])
+				if (fromIsContract) {
+					await VolumeService.updateVolume(`Contract.${from}`, amountValueInUSD, blockTimestamp)
+				}
+				if (toIsContract) {
+					await VolumeService.updateVolume(`Contract.${to}`, amountValueInUSD, blockTimestamp)
+				}
```

Comment:
I don't think we should do it this way. 7702 means any EOA can now have some code. We should instead check the transaction calldata and look at which contract the request/response is for, and use that to index transfer volumes.